omnibus/config/software/opensearch.rb

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright:: Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "opensearch"
+default_version "1.1.0"
```

Comment:
should this be 1.2.4?

Reply:
can we use omnibus-software/opensearch? https://github.com/chef/omnibus-software/blob/main/config/software/opensearch.rb#L18 so we can remove the dep on this file completely.

Reply:
There are some modifications to tools & plugins, Is it possible to change in omnibus-software directly ?
https://github.com/chef/chef-server/blob/zei/shahid/internal_opensearch/omnibus/config/software/opensearch.rb#L55=L62